roles/mariadb/tasks/main.yml

```diff
@@ -1,28 +1,40 @@
 ---
 # This playbook will install MariaDB and create db user and give permissions.
 
-- name: Install MariaDB package
-  yum: name={{ item }} state=installed
+- name: remove older versions of conflicting packages
+  package: name={{ item }} state=absent
   with_items:
-   - mariadb-server
-   - MySQL-python
-   - libselinux-python
-   - libsemanage-python
+    - mariadb-libs
 
-- name: Configure SELinux to start mysql on any port
-  seboolean: name=mysql_connect_any state=true persistent=yes
+- name: install MariaDB
+  package: name={{ item }} state=installed
+  with_items:
+   - mariadb101u-server
 
-- name: Create Mysql configuration file
-  template: src=my.cnf.j2 dest=/etc/my.cnf
+- name: copy MySQL configuration file
+  template:
+    src: my.cnf.j2
+    dest: /etc/my.cnf
   notify:
   - restart mariadb
 
-- name: Create MariaDB log file
-  file: path=/var/log/mysqld.log state=touch owner=mysql group=mysql mode=0775
+- name: create MariaDB log file
+  file:
+    path: /var/log/mysqld.log
+    state: touch
+    owner: mysql
+    group: mysql
+    mode: 0775
```

Comment:
Do you want 755 or 644 for the log file? I'm not sure what permissions this specific log file needs.